docs/docs/policies.mdx

```diff
@@ -84,10 +84,10 @@ Doing so can lead to unexpected and undesired bot behavior.
 
 The Transformer Embedding Dialogue (TED) Policy is
 a multi-task architecture for next action prediction and entity
-recognition. The architecture several transformer encoders which are shared for both tasks.
+recognition. The architecture consists of several transformer encoders which are shared for both tasks.
 A sequence of entity labels is predicted through a Conditional Random Field (CRF) tagging layer on top of the
 user sequence transformer encoder output corresponding to the input sequence of tokens.
-For the next action prediction the dialogue transformer encoder output and system action labels are embedded into a
+For the next action prediction, the dialogue transformer encoder output and system action labels are embedded into a
```

Comment:
"Dialogue transformer encoder output and system action labels" is also a bit of a mouthful, though I'm not certain what would be the best way to change it. Any ideas?

Reply:
Hmm, what about "transformer encoding and action labels"

Reply:
I think my specific issue is that it feels a bit ambiguous at current - not totally clear on my first pass reading it if "system action labels" belong to the "dialogue transformer encoder". I think your suggestion is a little more clear in that regard but I'm wondering if we're losing important context by dropping the name of the specific transformer.

Perhaps "For the next action prediction, the system action labels and the output of the dialogue transformer encoder are embedded into a single semantic vector space"? Does that read more clearly without causing confusion between the two things?

Reply:
The action labels themselves are separate from the transformer encoding, but are related in the sense that labels predicted at state N are computed using the encoding derived from state N-1. 

Yeah dialogue can be left in to avoid confusion and avoid ambiguity since a different transformer architecture is used for DIET.

Reply:
It reads clearly with the "the" added 👍